src/System.Net.Sockets/tests/FunctionalTests/SendPacketsAsync.cs

```diff
@@ -16,8 +16,9 @@ public class SendPacketsAsync
         private readonly ITestOutputHelper _log;
 
         private IPAddress _serverAddress = IPAddress.IPv6Loopback;
-        // In the current directory
-        private const string TestFileName = "NCLTest.Socket.SendPacketsAsync.testpayload";
+        // Accessible directories for UWP app:
+        // C:\Users\<UserName>\AppData\Local\Packages\<ApplicationPackageName>\
+        private string TestFileName = Environment.GetEnvironmentVariable("LocalAppData") + @"\NCLTest.Socket.SendPacketsAsync.testpayload";
```

Comment:
@davidsh New change passed on non-UWP Windows (netcore & netfx). I will wait for *Nix result in CI.

>Environment.SpecialFolder is not working right in Appcontainer right now

Yes, and I tested on my machine, Environment.SpecialFolder.LocalApplicationData doesn't work on netcore & netfx mode as well, for the same reason.